services/frontend/src/components/AnalysisResultTable.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="analysis-table-box">
+    <table> 
+      <tr>
+        <th>Numer analizy</th>
+        <th>Pesel/Nr Paszportu</th>
+        <th>Imię</th>
+        <th>Nazwisko</th>
+        <th>Data badania</th>
+        <th>Wynik analizy</th>
+      </tr>  
+      <tr 
+        v-bind:key="record.analysisId" 
+        v-bind:class="{'table-record-even': index % 2 === 0, 'table-record-odd': index % 2 === 1 }"  
+        v-for="(record, index) in analysisData"
+      >
+        <td>{{record.analysisId}}</td>
+        <td>{{'' + record.patientId + ' ' + record.patientIdType}}</td>
+        <td>{{record.patientName}}</td>
+        <td>{{record.patientSurname}}</td>
+        <td>{{record.date}}</td>
+        <td>
+          <a class="link" v-bind:href="record.analysisHyperlink">
+            Kliknij by pobrać wynik 🗃️
+          </a>
+        </td>
+      </tr>
+    </table>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'AnalysisResultTable',
+  props: ['analysisData']
+}
+</script>
+
+<style scoped>
+.analysis-table-box {
+  background-color: var(--color-sector);
+  border-radius: 3px;
+  padding-block: 40px;
+  padding-inline: 50px;
+  width: -webkit-fit-content;
+  height: -webkit-fit-content;
+  width: -moz-fit-content;
+  height: -moz-fit-content;
```

Comment:
Looks like a code smell, especially since there is no default value for non webkit/moz engines.
Try testing the solutions mentioned [here](https://stackoverflow.com/questions/7839164/is-there-a-css-cross-browser-value-for-width-moz-fit-content).

Reply:
That is some leftover trash from coppying style from my test project